tests/test_pipeline_images.py

```diff
@@ -79,28 +79,70 @@ def test_convert_image(self):
         SIZE = (100, 100)
         # straigh forward case: RGB and JPEG
         COLOUR = (0, 127, 255)
-        im = _create_image('JPEG', 'RGB', SIZE, COLOUR)
-        converted, _ = self.pipeline.convert_image(im)
+        im, buf = _create_image('JPEG', 'RGB', SIZE, COLOUR)
+        converted, converted_buf = self.pipeline.convert_image(im, buf)
```

Comment:
Purely :lipstick: aesthetic, so feel free to ignore it: since the `converted_buf` is not used, it may make sense not to name it (also applies to other changes):

```suggestion
        converted, _ = self.pipeline.convert_image(im, buf)
```